### ISSUE TITLE
Petronius only wrote the Satyrica

### ISSUE THREAD

**Issue by ThomasK81** [PerseusDL/catalog_data#57]
Could we merge

phi0972.phi001p
phi0972.phi1p
phi0972.phi001f
phi0972.phi1f
phi0972.phi001

to phi0972.phi001, please?

**Comment by AlisonBabeu** [comment]
So the whole challenge with this identifier request is that the current state of Petronius identifiers was created in order to support the fact that the Petronius texts, as they exist in the current P4 digital library, were split along the lines of how they appeared in the 1913 Loeb-Heseltine edition (https://archive.org/details/petronius00petruoft),  which split the works into the Satyricon, Fragments and Poems (http://www.perseus.tufts.edu/hopper/collection?collection=Perseus%3Acorpus%3Aperseus%2Cauthor%2CPetronius) , all of which are essentially, the Satyricon. These three existing "works" in Perseus all have existing CTS-URNs based off of these identifiers, so when I came across other editions of "Poems" and "Fragments", I reused these identifiers to remain consistent with how the work appeared in Perseus, although for some reasons they ended up being listed as phi0972.phi1p, and phi0972.phi1f. Due to the existence of CTSURNs in P4 that make use of the split identifiers, I'm not entirely sure how to proceed.

**Comment by ThomasK81** [comment]
I think this will be the case quite often. I started to look through all Latin texts, and you can see the paper-format history in a lot of them. I assume this will be similar for the Greek texts.
Here is my offer, I already did an edition of chapters 1-15 and some fragments of the Satyrica. I am reworking the rest of the Satyrica bit by bit, so we have a new Perseus/OGL produced and edited Petronius edition with commentary. Does this sound like a plan?
I have already changed the chapters in the whole Petronius text and Annette is currently putting in the paragraphs on the basis the 18th century Anton edition. I will produce an in progress edition that we can publish and update on Perseus. What do you think?

**Comment by perseuscatalog** [comment]
I really need a better grasp of what's happening on the OGL side....



On Aug 4, 2015, at 9:45 PM, Alison Babeu <notifications@github.com<mailto:notifications@github.com>> wrote:


So the whole challenge with this identifier request is that the current state of Petronius identifiers was created in order to support the fact that the Petronius texts, as they exist in the current P4 digital library, were split along the lines of how they appeared in the 1913 Loeb-Heseltine edition (https://archive.org/details/petronius00petruoft), which split the works into the Satyricon, Fragments and Poems (http://www.perseus.tufts.edu/hopper/collection?collection=Perseus%3Acorpus%3Aperseus%2Cauthor%2CPetronius) , all of which are essentially, the Satyricon. These three existing "works" in Perseus all have existing CTS-URNs based off of these identifiers, so when I came across other editions of "Poems" and "Fragments", I reused these identifiers to remain consistent with how the work appe ared in Perseus, although for some reasons they ended up being listed as phi0972.phi1p, and phi0972.phi1f. Due to the existence of CTSURNs in P4 that make use of the split identifiers, I'm not entirely sure how to proceed.

**Comment by ThomasK81** [comment]
Should we have a hangout tomorrow? I could do 4pm Boston time that would be
8am NZT. Cheers, Thomas

Dr Thomas Koentges
Akademischer Assistent (Assistant Professor)
Digital Humanities
Universität Leipzig

thomas.koentges@uni-leipzig.de
thomas.koentges@googlemail.com

2015-08-05 14:38 GMT+12:00 perseuscatalog <notifications@github.com>:

> I really need a better grasp of what's happening on the OGL side....
>
>
>
> On Aug 4, 2015, at 9:45 PM, Alison Babeu <notifications@github.com<mailto:

**Comment by gregorycrane** [comment]
I think that going forward we will need to do something along the following lines often. A problem may be managing the the fact that we will have multiple citation schemes in one work.



Could we merge

phi0972.phi001p
phi0972.phi1p
phi0972.phi001f
phi0972.phi1f
phi0972.phi001

to phi0972.phi001, please?

**Comment by balmas** [comment]
fwiw I don't think we should make any decisions based upon supporting P4.  If, in the catalog data, we retire and redirect the invalid work urns (e.g. phi0972.phi1f) to the correct ones, eventually it should work out.

**Comment by AlisonBabeu** [comment]
I have to agree. The Petronius decision is a few years old at this point. In terms of the catalog data that doesn't point to Perseus editions, it is a pretty simple matter to retire old work URNs and redirect the various URNs to all new phi0972.phi001 versions. My most pressing question I guess was what do we do in terms of the Perseus CTS-URNs. Would we want to merge all three unique TEI-XML documents Satyricon, Poems and Fragments into one document and then just have them all under the top level URNs of  urn:cts:latinLit:phi0972.phi001.perseus-lat1 and urn:cts:latinLit:phi0972.phi001.perseus-eng1.

**Comment by balmas** [comment]
@AlisonBabeu yes I think if we have decided they are one work, then merging is consistent with the approach we have taken for the multi-volume works.
